### ISSUE TITLE
[Bug]: TalkBack does not read the back button for add-ons pop up

### ISSUE THREAD

**Issue by mariansima** [mozilla-mobile/fenix#21471]
### Steps to reproduce

1. Have at least one add-on installed
2. Turn TalkBack (or the available screen reader) on
3. Open the 3-dot menu > Add-ons
4. Select the first item in the list 

### Expected behaviour

TalkBack reads out the description of the item (navigate up button) and available actions (navigate up button - double tap to activate)

### Actual behaviour

The title is read out (Add-ons-double tap to activate) without describing the actual effect of the button (navigate up button)

### Device name

Samsung Galaxy A8

### Android version

Android 9

### Firefox release type

Firefox Nightly

### Firefox version

94.0.a1 2021-09-23

### Additional information

Also occurs on 93.0.0-beta.2 2021-09-13. The issue does not occur for RC 92.1.1 2021-09-04 because the button opens the add-ons manager

https://user-images.githubusercontent.com/89511597/134507006-ef94af33-8d29-45fe-b8e1-03fb60dd78cc.mp4

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-17993)

**Comment by sflorean** [comment]
This is still reproducible on latest Nightly builds.
